src/Model/Table/PermissionsTable.php

```diff
@@ -64,11 +64,11 @@ public function check($aro, $aco, $action = '*')
         $aroPath = $this->Aro->node($aro);
         $acoPath = $this->Aco->node($aco);
 
-        if (!$aroPath || !$acoPath) {
+        if (!$aroPath) {
```

Comment:
@markstory There is a test just after this one for aco :
https://github.com/cakephp/acl/pull/27/files#diff-b66e5b9be2d32eff472c66f61b4dda36R81